lib/galaxy/datatypes/converters/bgzip.py

```diff
@@ -6,11 +6,12 @@
 usage: %prog in_file out_file
 """
 import optparse
-import subprocess
 import tempfile
 
 import pysam
 
+from galaxy.util import commands
```

Comment:
I don't think we'll want to add another dependency here, this file is run by external tools.

Reply:
Should I: 

- just drop the change? or
- add unicodify here... but I guess this also comes with another dependency or
- add a python2 requirement to the tools using this script (I guess they could need it anyway)

Reply:
The converter works fine on python 3, I think the current code works fine on python 3 (we run converter tests now).
I wouldn't touch this, or if you do I would replace this file and just put the command in cheetah.

Reply:
> I wouldn't touch this, or if you do I would replace this file and just put the command in cheetah.

It should be possible by replacing `pysam.tabix_compress()` with `bgzip` from the `htslib` package.

Reply:
Then I will undo these changes. replacing pysam sounds like a separate PR -- what would be the advantage?

Reply:
> or if you do I would replace this file and just put the command in cheetah

since 4-5 tools use this python script it seems that this would result in quite a bit of code duplication.

Reply:
Yes, I think that's for another PR. I guess getting rid of the additional wrapper is the main advantage.

Reply:
> since 4-5 tools use this python script it seems that this would result in quite a bit of code duplication.

These don't need to be separate the way they are now. You can use one converter for many inputs.

Reply:
Made a note here https://github.com/galaxyproject/galaxy/issues/9672